src/database/firebase_list_factory.spec.ts

```diff
@@ -541,6 +541,22 @@ describe('FirebaseListFactory', () => {
       });
     });
 
+    it('should emit values in the correct zone', (done: any) => {
+      Zone.current.fork({
+        name: 'newZone'
+      })
+      .run(() => {
+        let observable = FirebaseListFactory(`${rootDatabaseUrl}/questions`);
+        ref.remove(() => {
```

Comment:
How could the zone be something else `newZone` here ?

I think you should:
- call subscribe from `newZone`,
- trigger the cb from any other zone,
- assert that cb in correct zone